css/css-grid/alignment/references/grid-baseline-align-001-ref.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<title>
+	Grid Item (First) Baseline Block-Axis Alignment: auto-height grid, auto row, auto items
+</title>
+<link rel="help" href="https://drafts.csswg.org/css-grid-1/#column-align">
+<link rel="help" href="https://www.w3.org/TR/css-grid-1/#algo-content">
+<link rel="help" href="https://www.w3.org/TR/css-align-3/#baseline-values">
+<link rel="match" href="references/grid-baseline-align-001-ref.html">
```

Comment:
lint is failing because of the leading `references/` here (we're already in that dir) but also if you remove that, the ref is pointing at itself, which is another lint error. Probably this was just copy-pasted and there shouldn't be a `rel="match"` here at all?